tests/test_scdataframe.py

```diff
@@ -0,0 +1,107 @@
+"""
+Tests cosmicqc SCDataFrame module
+"""
+
+import pathlib
+
+import pandas as pd
+from cosmicqc.scdataframe import SCDataFrame
+from pyarrow import parquet
+
+
+def test_SCDataFrame_with_dataframe(
```

Comment:
Best practices curiosity question: Is it preferred to test each of the individual file suffixes separately? My intuition here is to test each in the same function but loop through having the same assertions

Reply:
Thanks for this thought - good idea! I think these could be moved to a single test function to simplify as you suggested. Generally I worry about testing isolation for these scenarios where it'd be easy to miss a keystroke or accidentally copy + paste into the wrong scenario. Just the same, we can keep the formatting similar but without the function definitions here, adding a comment to keep things split up.